cea/technologies/heat_exchangers.py

```diff
@@ -59,13 +59,13 @@ def calc_Cinv_HEX(Q_design_W, locator, config, technology_type):
         InvC = Inv_a + Inv_b * (Q_design_W) ** Inv_c + (Inv_d + Inv_e * Q_design_W) * log(Q_design_W)
 
         Capex_a_HEX_USD = InvC * (Inv_IR) * (1 + Inv_IR) ** Inv_LT / ((1 + Inv_IR) ** Inv_LT - 1)
```

Comment:
just noticed that we seem to be using the same formula for calculating opex and capex for different technologies. maybe that could be refactored to a method? pass in the variables / maybe even the row in the dataframe or whatever and out come opex/capex?

Reply:
the calculation is made for one hourly data point at the time. let's create a separate issue to refactor that

Reply:
see #2189